src/core/display/Transform.js

```diff
@@ -156,7 +148,6 @@ export default class Transform extends TransformBase
     set rotation(value)
     {
         this._rotation = value;
-        this._sr = Math.sin(value);
-        this._cr = Math.cos(value);
+        this.updateSkew();
     }
```

Comment:
lets do it in next iteration. I also want to add "shear" as the real field , rather than "skew"